src/connectors/dispatcher/impl.js

```diff
@@ -99,5 +100,43 @@ module.exports = new class extends Connector {
         return result;
     }
 
+    async getPlaybookRunRecipientStatus (dispatcherStatusRequest) {
+        const uri = new URI(host);
+        uri.segment('internal');
+        uri.segment('v2');
+        uri.segment('recipients');
+        uri.segment('status');
+
+        const options = {
+            uri: uri.toString(),
+            method: 'POST',
+            json: true,
+            rejectUnauthorized: !insecure,
+            headers: this.getForwardedHeaders(),
+            body: dispatcherStatusRequest
+        };
+
+        // This header should be sent to the playbook dispatcher for each internal request.
+        if (auth) {
+            options.headers = {
+                Authorization: `PSK ${auth}`
+            };
+        }
+
+        const result = await this.doHttp (options, false, this.postRunRequests);
```

Comment:
Could there be a case here were you might need to fetch more than one "page" of data?